lib/matplotlib/figure.py

```diff
@@ -1647,7 +1469,7 @@ def _gci(self):
         ``gci`` (get current image).
         """
         # Look first for an image in the current Axes:
-        cax = self._axstack.current_key_axes()[1]
+        cax = self._axstack()
         if cax is None:
             return None
         im = cax._gci()
```

Comment:
Can be simplified to:
```suggestion
        if self._axstack.empty():
            return None
        im = self._axstack()._gci()
```